next.config.js

```
@@ -11,115 +11,147 @@ const withPWA = NextPWA({
+  // Image optimization settings
+  images: {
+    formats: ['image/avif', 'image/webp'],
+    deviceSizes: [640, 750, 828, 1080, 1200, 1920, 2048, 3840],
+    imageSizes: [16, 32, 48, 64, 96, 128, 256, 384],
+    minimumCacheTTL: 60,
+    dangerouslyAllowSVG: true,
+    contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
+  },
+
+  // Webpack configuration
+  webpack: (config, { dev, isServer, defaultLoaders }) => {
```

ESLint

[error] 'defaultLoaders' is defined but never used.

(@typescript-eslint/no-unused-vars)